projects/plugins/boost/changelog/e2e-remove-duplicate-fixture

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
Why minor rather than patch? This only affects testing, it doesn't change the plugin at all.
```suggestion
Significance: patch
```
That'll also save having to do all the version bumps below.

Reply:
> Why minor rather than patch? This only affects testing, it doesn't change the plugin at all.

Fixed with [42cb0e8](https://github.com/Automattic/jetpack/pull/23703/commits/42cb0e8a753a0e38408ab2607776626f0bdc524d)

> That'll also save having to do all the version bumps below.

TIL